go/v1beta1/server/server.go

```diff
@@ -0,0 +1,247 @@
+// Copyright 2019 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
```

Comment:
Please add a comment for this package.

Reply:
Amended.

Reply:
Package comments should be in the form of:

Package [PACKAGE_NAME]...

So:

Package server includes all generic...

See:
https://blog.golang.org/godoc-documenting-go-code

this is primarily so it's easier to search Godoc.